test/feature/reverse_routing.feature

```diff
@@ -38,18 +39,28 @@ Feature: reverse routing
     Then I have a routing helper "<HELPER>" that returns "<VALUE>"
 
     Examples:
-      | HELPER                         | VALUE           |
-      | res.locals.paths.users()       | /users          |
-      | res.locals.paths.user(123)     | /users/123      |
-      | res.locals.paths.newUser()     | /users/new      |
-      | res.locals.paths.editUser(123) | /users/123/edit |
+      | HELPER                                    | VALUE           |
+      | res.locals.routes.users()                 | /users          |
+      | res.locals.routes.user(123)               | /users/123      |
+      | res.locals.routes.newUser()               | /users/new      |
+      | res.locals.routes.editUser(123)           | /users/123/edit |
+      | res.locals.routes.updateUser(123)         | /users/123      |
+      | res.locals.routes.createUser()            | /users          |
+      | res.locals.routes.destroyUser(123)        | /users/123      |
+      | res.locals.routes.users().method          | GET             |
```

Comment:
`Then I have a routing helper "<HELPER>" that returns "<VALUE>" and the HTTP verb "<METHOD>"`

I think this is clear enough. The exact usage is explained in the readme.